notebooks/en/enterprise_dedicated_endpoints.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<blockquote>Now let's create an endpoint for a vision language model (VLM). VLMs are very similar to LLMs, with the difference of taking text and images inputs. These models autoregressively generate text, just like a standard LLM. VLMs can tackle many tasks varying from visual question answering to document understanding. For this example, we will use <a href="https://huggingface.co/blog/idefics2" target="_blank">Idefics2</a>, a powerful 8B parameter VLM.  </blockquote><p>I feel like document reasoning is a very cool industry use case for multimodal so I thought I'd add it, IDEFICS2 has good doc capabilities</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>